bundles/CoreBundle/Command/Document/GeneratePagePreviews.php

```diff
@@ -43,9 +44,60 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Prefix for the document path, eg. https://example.com, if not specified, Pimcore will try use the main domain from system settings.',
                 null
+            )
+            ->addOption(
+                'parent',
+                'p',
+                InputOption::VALUE_OPTIONAL,
+                'Only create preview of objects in this folder (ID|path).'
+            )
+            ->addOption(
+                'regex',
+                'r',
```

Comment:
I think it is more clear
```suggestion
                'exclude-patterns',
                'e',
```